junifer/external/nilearn/tests/test_junifer_nifti_spheres_masker.py

```diff
@@ -0,0 +1,299 @@
+"""Test custom nilearn.maskers.NiftiSpheresMasker."""
+
+# Authors: Synchon Mandal <s.mandal@fz-juelich.de>
```

Comment:
Same here, how much is it nilearn's code?

Reply:
The whole of it. I removed the tests for `inverse_transform`.

Reply:
then keep nilearn's header then.

Reply:
This will also need the license, I believe?

Reply:
Yes.